CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
+* container: promoted `node_pool_auto_config` field in `google_container_cluster` from beta provider to GA provider. (ga) ([#6299](https://github.com/hashicorp/terraform-provider-google-beta/pull/6299))
+* container: promoted field `placement_policy.tpu_topology` in resource `google_container_node_pool` to GA ([#6286](https://github.com/hashicorp/terraform-provider-google-beta/pull/6286))
+* containeraws: added support for `auto_repair` in `container_aws_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* containerazure: added support for `auto_repair` in `container_azure_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* filestore: added support for the `"ZONAL"` value to `tier` in `google_filestore_instance` ([#6303](https://github.com/hashicorp/terraform-provider-google-beta/pull/6303))
+* firestore: added `delete_protection_state` field to `google_firestore_database` resource. ([#6295](https://github.com/hashicorp/terraform-provider-google-beta/pull/6295))
+* identityplatform: added `sign-in` field to `google_identity_platform_config` resource ([#6293](https://github.com/hashicorp/terraform-provider-google-beta/pull/6293))
+* networkconnectivity: added support for `linked_vpc_network` in `network_connectivity_spoke` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* networkservices: increased default timeout for `google_network_services_edge_cache_origin` to 120m from 60m ([#6275](https://github.com/hashicorp/terraform-provider-google-beta/pull/6275))
+* networkservices: increased default timeout for `google_network_services_edge_cache_service` to 60m from 30m ([#6281](https://github.com/hashicorp/terraform-provider-google-beta/pull/6281))
+* secretmanager: added `is_secret_data_base64` field to `google_secret_manager_secret_version` resource ([#6273](https://github.com/hashicorp/terraform-provider-google-beta/pull/6273))
+* workstations: added `env` field to `google_workstations_workstation` resource (beta) ([#6258](https://github.com/hashicorp/terraform-provider-google-beta/pull/6258))
+
+BUG FIXES:
+* bigquery: update documentation for `google_bigquery_table.time_partitioning.expiration_ms` ([#6290](https://github.com/hashicorp/terraform-provider-google-beta/pull/6290))
+* bigtable: added a read timeout to `google_bigtable_instance` ([#6276](https://github.com/hashicorp/terraform-provider-google-beta/pull/6276))
+* bigtable: improved regional reliability when instance overlaps a downed region in the resource `google_bigtable_instance` ([#6313](https://github.com/hashicorp/terraform-provider-google-beta/pull/6313))
+* eventarc: field `event_data_content_type` in `eventarc_trigger` now marked as computed ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```

Comment:
this looks like a breaking change; just want to double-check it's fine in a minor release?

Reply:
```suggestion
* eventarc: field `event_data_content_type` in `google_eventarc_trigger` now marked as computed ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```

Reply:
I believe the field went from optional to optional+computed.